BD-3715 No entry or exit events for split steps (#8665)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md

CanvasStep, Progression
{% endapitags %}

> This event occurs when a user progresses through a step in a Canvas with some outcome. Currently, only split steps (Audience Paths, Decision Split, Action Paths, Experiment) and advance outcomes generate step progression events. There are no entry or exit events for split steps.

Author: ```suggestion
This event occurs when a user progresses through a step in a Canvas with some outcome. Note that this event doesn't occur when steps are entered or exited. Currently, only split steps (Audience Paths, Decision Split, Action Paths, Experiment) and advance outcomes generate step progression events.
```
